rust/worker/src/execution/operators/merge_knn_results.rs

```diff
@@ -11,13 +11,18 @@ use thiserror::Error;
 #[derive(Debug)]
 pub struct MergeKnnResultsOperator {}
 
+#[derive(Debug)]
+pub struct BruteForceResult {
+    pub user_ids: Vec<String>,
+    pub distances: Vec<f32>,
+    pub vectors: Vec<Vec<f32>>,
+}
+
 #[derive(Debug)]
 pub struct MergeKnnResultsOperatorInput {
```

Comment:
review: wondering if we should rename this as it does work beyond just merging? (i.e. we still need to run it even if there's no brute force results)

Reply:
makes sense to me

Reply:
decided to keep the current name for now, can't think of an alternative that isn't a mouthful